### PR TITLE
[Feature] [Louie] UI changes and Specific Product Navigation animation

### PR DESCRIPTION
**Pull request by cezro**
**Clickup Task:**
https://app.clickup.com/t/865cr46kg

**Description:**
Slight UI changes to specific product page and transition animation

**Images/Videos:**
https://github.com/ezerssss/unihub/assets/90561528/143be1dc-47c1-4fd8-879d-efb8256267e4

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -49,7 +49,13 @@ export default function App() {
               <Stack.Screen component={Sell} name={Routes.SELL} />
               <Stack.Screen component={EditSell} name={Routes.EDIT_SELL} />
               <Stack.Screen component={Chat} name={Routes.CHAT} />
-              <Stack.Screen component={SpecificProduct} name={Routes.PRODUCT} />
+              <Stack.Screen
+                component={SpecificProduct}
+                name={Routes.PRODUCT}
+                options={{
+                  animation: 'simple_push',
+                }}
```
Comment (ezerssss) [inline App.tsx:57]: no need for `animationTypeForReplace: 'push'`?

**Review by ezerssss — APPROVED** [review]

File: App.tsx (hunk 1 above)
Comment (cezro) [reply App.tsx:57]: hmm not actually sure what that does, what kind of issues could it do?

Comment (ezerssss) [reply App.tsx:57]: well for mine it didnt work, w/o it, pero if it does work then dats alright

just have to let QA figure it out HAHA

Comment (cezro) [reply App.tsx:57]: might be an android issue, ill just resolve it if ever lmao

**Review by jedmamosto — CHANGES_REQUESTED** [review]
Animation looks good! BG color implementation is there but you forgot to implement changes for the margins: 

**WHAT IT LOOKS LIKE:** 

image


**SHOULD BE: (it aligns vertically with the image)** 

image

image

**Comment by jedmamosto** [comment]
Oof you have merge conflicts Lou maybe have that resolves first

As for the requested changes, all good! Good job

**Review by jedmamosto — APPROVED** [review]
looks good! approved

**Review by caspted — APPROVED** [review]
Good job on this task the animation looks good and the details are aligned in android

**Pull request merged**